doc/changelog.rst

```diff
@@ -10,6 +10,10 @@ Version 0.5
 
 **Changes**:
 
+* Moved function ``fatiando.utils.circular_points`` to
+  ``fatiando.gridder.circular_scatter`` module. Deleted the functions
+  `utils.connect_points` and `utils.random_points`. Unit tests are done.
+  (`PR 317  <https://github.com/fatiando/fatiando/pull/317>`__)
```

Comment:
@victortxa could you please change this to something like:

    Moved function ``fatiando.utils.circular_points`` to
    ``fatiando.gridder.circular_scatter`` module because ``gridder``
    is where point generation functions live. 
    The function now returns the x, y coordinate arrays instead of
    a list of points. You can transform x, y to points by
    ``points = numpy.transpose([x, y])``.
    Deleted the function  ``utils.connect_points`` because it was 
    only used in a single place and could be substituted by two 
    list comprehensions. Deleted function ``utils.random_points`` 
    because it has the exact same functionality as ``gridder.scatter``.